pkg/apis/servicecatalog/validation/serviceclass.go

```diff
@@ -30,7 +30,7 @@ import (
 // validateServiceClassName is the validation function for ServiceClass names.
 var validateServiceClassName = apivalidation.NameIsDNSSubdomain
 
-const guidFmt string = "[a-zA-Z0-9]([-a-zA-Z0-9]*[a-zA-Z0-9])?"
+const guidFmt string = "[a-zA-Z0-9]([-a-zA-Z0-9.]*[a-zA-Z0-9])?"
```

Comment:
I would prefer not to drop them all, or we will never get a clear picture of what kind of stuff people return in this field.